apps/server/src/routes/chat.ts

```
@@ -1,112 +1,471 @@
+  async onChatMessage(onFinish: StreamTextOnFinishCallback<{}>) {
+    if (!this.driver) {
+      return new Response('Unauthorized', { status: 401 });
+    }
+    return this.getDataStreamResponse(onFinish);
   }
+}
 
-  const _conn = await getActiveConnection().catch((err) => {
-    console.error('Error in getActiveConnection:', err);
-    throw c.json({ error: 'Failed to get active connection' }, 500);
+export class ZeroMCP extends McpAgent<typeof env, {}, { cookie: string }> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)